pitop/pma/servo_motor.py

```diff
@@ -118,16 +111,6 @@ def set_target_angle(self, angle, speed=50.0):
         self._controller.set_target_angle(angle + self._zero_point, speed)
         self.__has_set_angle = True
 
-    def get_target_speed(self):
```

Comment:
Why is this function deleted? I think that `target_angle` should be a property with a getter and setter

Reply:
We can keep it as a class variable if you want to leave it here, but we won't be reading that value from the MCU anymore. I just assumed that the user code would have access to their own target value that they set